### PR TITLE
(gom-player) New Package

### PR DESCRIPTION
**Pull request by AdmiringWorm**

### CODE REVIEW

File: automatic/gom-player/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿$ErrorActionPreference = 'Stop';
```
Comment (pascalberger) [inline automatic/gom-player/tools/chocolateyinstall.ps1]: `;` not required

Comment (AdmiringWorm) [inline automatic/gom-player/tools/chocolateyinstall.ps1]: But I like the `;` 😛 
But honestly, it's just a habit from C#, I'm more surprised when I don't add it.
I'll remove it before merging though.

Comment (pascalberger) [inline automatic/gom-player/tools/chocolateyinstall.ps1]: Happens to me too all the time :)

**Comment by AdmiringWorm** [comment]
I tried pushing this package now before merging it, but somehow I missed that it already existed (in rejected state and all version unlisted).

@gep13 Possible to perhaps replace the existing maintainer with the chocolatey user (the package was submitted in 2015 and rejected in january 2016)

**Comment by pascalberger** [comment]
@AdmiringWorm Done

**Comment by gep13** [comment]
@AdmiringWorm @pascalberger current version has been unrejected, ready for new submission.

**Comment by AdmiringWorm** [comment]
@gep13 sorry, I didn't mean for it to be unrejected. The latest version is higher than the rejected version, and they only provide the latest version on their webpage (No distribution rights either).

The reason I mentioned you and didn't merge the PR was because I was unsure if it was the correct way with replacing the old maintainer with the chocolatey user.

**Comment by gep13** [comment]
>@AdmiringWorm said...
because I was unsure if it was the correct way with replacing the old maintainer with the chocolatey user.

No worries.  Re-rejected :-)

Given the length of elapsed time from the rejected package, adding the chocolatey user and taking over the package is a perfectly valid route to take.  And looks like @pascalberger is in a position to help with this task as well 👍

**Comment by AdmiringWorm** [comment]
@gep13 I thought so, just wanted to make sure.

**Pull request merged**